environments/staging/public.yml

```diff
@@ -60,10 +60,19 @@ localsettings:
   BANK_NAME: "RBS Citizens N.A."
   BANK_SWIFT_CODE: 'CTZIUS33'
   CELERY_PERIODIC_QUEUE: 'celery_null'
+  COMMCARE_NAME:
+    'reach.commcarehq.org': 'REACH'
+    'default': 'CommCare'
+  COMMCARE_HQ_NAME:
+    'reach.commcarehq.org': 'REACH HQ'
+    'default': 'CommCare HQ'
   COUCH_CACHE_DOCS: True
   COUCH_CACHE_VIEWS: True
   COUCH_PASSWORD: "{{ localsettings_private.COUCH_PASSWORD }}"
   COUCH_USERNAME: "{{ localsettings_private.COUCH_USERNAME }}"
+  CUSTOM_LANDING_TEMPLATE:
+    'reach.commcarehq.org': 'icds/login.html'
```

Comment:
Just checking that we are reusing the icds templates here.

Reply:
Yeah. I'm doing this for testing since at this point we only have our default and ICDS login pages.

When I push this to TCL, then it will be reversed where reach is the commcare default theming and icds is their custom theme.In the future reach will probably have its own theming